core/src/main/scala/org/broadinstitute/dsde/rawls/metrics/RawlsInstrumented.scala

```diff
@@ -134,6 +179,22 @@ trait RawlsInstrumented extends DefaultInstrumented {
     status => builder
       .expand(WorkflowStatusMetric, status)
       .asCounter()
+
+  protected def httpRequestCounter(currentBuilder: Option[ExpandedMetricBuilder] = None): HttpRequest => Counter = {
```

Comment:
OK, makes sense. For my subsystem metrics I am passing in `httpRequestCounter(ExpandedMetricBuilder.expand(SubsystemMetricKey, Subsystems.Cromwell))` (for example) but it makese sense for it to be optional for your use case.